docs/constraints/TypeScript.md

```diff
@@ -11,7 +11,7 @@ These are the constraints that is applied to model naming. The `Rule key` is wha
 |NO_NUMBER_START_CHAR|No numbers as starting characters|Default behavior is pre pending `number_` in front of the first character|
 |NO_EMPTY_VALUE|No empty values|Default behavior is to use `empty` as name. |
 |NO_RESERVED_KEYWORDS|No reserved keywords|TypeScript has a list of reserved keywords ([see the full list here](../../src/generators/typescript/Constants.ts))|
-|NAMING_FORMATTER|Must be formatted equally|Model name is formatted using pascal case|
+|NAMING_FORMATTER|Must be formatted equally|Property name is formatted using pascal case|
```

Comment:
This section here is actually for the Model naming formatted 🙂 So this needs to be reverted. The other section is correct 👍 

Remember to propagate it across languages 🙂

Reply:
Hey @jonaslagoni, Thanks for the review! I made the necessary changes.